File: specification/specs/x_pack/machine_learning/close_job/CloseJobRequest.ts
@rest_spec_name("ml.close_job")
class CloseJobRequest extends RequestBase {
  path_parts?: {
    job_id: Id;
  }
  query_parameters?: {
    allow_no_jobs?: boolean;
    force?: boolean;
    timeout?: Time;
  }
  body?: {
  }
}

File: output/csharp/XPack/MachineLearning/CloseJob/CloseJobRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class CloseJobRequest : RequestBase {
		
		[DataMember(Name="allow_no_jobs")]
		public bool AllowNoJobs { get; set; }


		[DataMember(Name="force")]
		public bool Force { get; set; }


		[DataMember(Name="timeout")]
		public Time Timeout { get; set; }

	}
}
